*** tests/A-01-02-M-ok.out	2024-10-06 21:01:08.179654659 +0100
--- tests/A-01-02-M-ok.outhyp	2024-10-07 01:23:26.001748948 +0100
***************
*** 39,45 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
--- 39,45 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
***************
*** 51,54 ****
--- 51,55 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: 
\ Sem newline no fim do ficheiro
